### PR TITLE
New Ghostery assets

### LINKED ISSUE #219: Windows installer icon branding
The changes in #119 did not update the icon. Probably we changed the wrong .ico file.

### PR DESCRIPTION
**Pull request by chrmod**
Additonllay it fixes the problem with fern not commiting the windows installer icon. 

fix #219

### CODE REVIEW

**Comment by chrmod** [comment]
Still missing few files to match the correct format:
```
% diff -i <(cd ghostery; find . -print0 | xargs -0 file --) <(cd official; find . -print0 | xargs -0 file --)
5c5
< ./firefox.icns:                           Mac OS X icon, 226171 bytes, "ic12" type
---
> ./firefox.icns:                           Mac OS X icon, 1021785 bytes, "s8mk" type
17d16
< ./setup.ico:                              MS Windows icon resource - 6 icons, 16x16, 32 bits/pixel, 32x32, 32 bits/pixel
21,23c20,23
< ./stubinstaller/bgstub.jpg:               JPEG image data, JFIF standard 1.01, aspect ratio, density 72x72, segment length 16, Exif Standard: [TIFF image data, big-endian, direntries=1], progressive, precision 8, 1344x822, components 3
< ./VisualElements_70.png:                  PNG image data, 176 x 176, 8-bit/color RGBA, non-interlaced
< ./document.icns:                          Mac OS X icon, 245212 bytes, "ic12" type
---
> ./stubinstaller/bgstub.jpg:               JPEG image data, JFIF standard 1.01, aspect ratio, density 1x1, segment length 16, progressive, precision 8, 1344x822, components 3
> ./LICENSE:                                ASCII text
> ./VisualElements_70.png:                  PNG image data, 126 x 126, 8-bit/color RGBA, non-interlaced
> ./document.icns:                          Mac OS X icon, 501145 bytes, "s8mk" type
35a36
> ./content/about-logo.svg:                 SVG Scalable Vector Graphics image
38c39
< ./content/about.png:                      PNG image data, 300 x 236, 8-bit/color RGBA, non-interlaced
---
> ./content/about.png:                      PNG image data, 300 x 236, 8-bit/color RGB, non-interlaced
43c44
< ./disk.icns:                              Mac OS X icon, 55996 bytes, "ic09" type
---
> ./disk.icns:                              Mac OS X icon, 1525764 bytes, "s8mk" type
45c46
< ./background.png:                         PNG image data, 512 x 320, 8-bit/color RGB, non-interlaced
---
> ./background.png:                         PNG image data, 1152 x 768, 8-bit/color RGB, non-interlaced
49c50
< ./wizWatermark.bmp:                       PC bitmap, Windows 95/NT4 and newer format, 164 x 314 x 24
---
> ./wizWatermark.bmp:                       PC bitmap, Windows 3.x format, 164 x 314 x 24
```

**Comment by chrmod** [comment]
@remusao it would be great if `skip` option for custom patches compare the files that need to be copied. 

There was a bug in ghostery-branding custom patch - `fern reset` removed files from `browser/brandings/ghostery` but the folder was still there so subsequent run of `fern import-patches` was skipping ghostery-branding.

**Comment by remusao** [comment]
> @remusao it would be great if `skip` option for custom patches compare the files that need to be copied.
> 
> There was a bug in ghostery-branding custom patch - `fern reset` removed files from `browser/brandings/ghostery` but the folder was still there so subsequent run of `fern import-patches` was skipping ghostery-branding.

Mmh ok, but why was there a folder named `browser/brandings/ghostery` there? For now fern.js assume that the workspace is mostly clean (at least when it comes to files/folders which are managed by fern itself).

**Comment by chrmod** [comment]
> > @remusao it would be great if `skip` option for custom patches compare the files that need to be copied.
> > There was a bug in ghostery-branding custom patch - `fern reset` removed files from `browser/brandings/ghostery` but the folder was still there so subsequent run of `fern import-patches` was skipping ghostery-branding.
> 
> Mmh ok, but why was there a folder named `browser/brandings/ghostery` there? For now fern.js assume that the workspace is mostly clean (at least when it comes to files/folders which are managed by fern itself).

Yeah, there was `browser/brandings/ghostery/locales` folder present after `fern reset`

**Comment by remusao** [comment]
Unless it is placed by something outside of fern's control I don't see how it's happening. At least I cannot reproduce locally when running: `fern.js reset && fern.js import-patches && fern.js reset` then `mozilla-release` is always clean.

File: fern/patches/ghostery-branding.js [hunk 1]
```diff
@@ -3,41 +3,49 @@ const path = require("path");
...
+    ...windowsInstallerIconPathComponents
   );
 }
 
 module.exports = {
   name: "Setup Ghostery branding",
-  paths: ["browser/branding/ghostery"],
-  skip: async () => folderExists(await getPathToGhosteryBranding()),
+  paths: ["browser/branding/ghostery", path.join(...windowsInstallerIconPathComponents)],
+  skip: async () => false,
```
Comment (remusao) [inline fern/patches/ghostery-branding.js:40]: If it already exists, will `fsExtra.copy` succeed? (I would expect to have an exception raised?

Comment (chrmod) [reply fern/patches/ghostery-branding.js:40]: Does not complain to me

**Comment by chrmod** [comment]
The repo was clean, but the folder existed as git does not track folders.

**Comment by remusao** [comment]
> The repo was clean, but the folder existed as git does not track folders.

I did not understand where this folder came from but if it works for you now I guess that's fine.

**Comment by chrmod** [comment]
.DS_Store files are to blame it looks
```
./fern.js reset
find mozilla-release/browser/branding/ghostery
```

```
mozilla-release/browser/branding/ghostery
mozilla-release/browser/branding/ghostery/.DS_Store
mozilla-release/browser/branding/ghostery/locales
mozilla-release/browser/branding/ghostery/locales/.DS_Store
```

**Comment by chrmod** [comment]
@remusao should we add other step to fern reset to clean those things?

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**